pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.bullhorn</groupId>
     <artifactId>sdk-rest</artifactId>
-    <version>1.4.50</version>
+    <version>1.4.51</version>
```

Comment:
This probably needs to be updated to 1.4.53

Reply:
@taswartz you're right here, too. But I was thinking this was going to be updated to 2.0 when we release the java17 branch anyway.

Reply:
I would actually prefer revert this small change so we delegate the version bump to the java17 release

Reply:
I agree!